prelude/__internal__/List.mad

```diff
@@ -168,7 +170,7 @@ export last = (xs) => #-{
 }-#
 
 /**
- * Returns a Maybe containing either a Just with the first element or a Nothing if the list is empty.
+ * Safely access the first value in a List (wrapped in Just), or Nothing.
```

Comment:
Same as above, what do you think?

Reply:
Well, also we have the examples though, not sure. Happy to hear your thought before we decide.

Reply:
Also amended this one.